contracts/WakuRlnRegistry.sol

```diff
@@ -56,12 +55,28 @@ contract WakuRlnRegistry is Ownable {
                     assembly {
                         revert(add(32, err), mload(err))
                     }
+                    // when there are no further storage contracts available, revert
+                } else if (usingStorageIndex + 1 >= nextStorageIndex) {
+                    revert NoStorageContractAvailable();
                 }
                 usingStorageIndex += 1;
             }
         }
     }
 
+    function register(uint16 storageIndex, uint256[] calldata commitments) external payable {
+        if (storageIndex >= nextStorageIndex) revert NoStorageContractAvailable();
+        WakuRln(storages[storageIndex]).register(commitments);
+    }
+
+    function register(uint16 storageIndex, uint256 commitment) external payable {
+        if (storageIndex >= nextStorageIndex) revert NoStorageContractAvailable();
```

Comment:
Should we make this a modifier?

Reply:
Addressed in [002c0f1](https://github.com/waku-org/waku-rln-contract/pull/3/commits/002c0f1213ac0bd660c77cb838484a31e1cb8edc) for the `usingStorageIndex` variable

Reply:
Hmm, should I extend the modifier to accept a value, so that it can be used for the `register` functions accepting a `storageIndex` ?

Reply:
I think this can be done once there's indeed a need for it. No strong opinions :)

Reply:
> Hmm, should I extend the modifier to accept a value, so that it can be used for the register functions accepting a storageIndex ?

My apologies. This is actually what I was looking for originally. Or rather, I was looking for a modifier that does the 
```solidity
if (storageIndex >= nextStorageIndex) revert NoStorageContractAvailable();
```

check. 


Just noticed that `register()` still doesn't have a modifier here and only now realized that your [comment(https://github.com/waku-org/waku-rln-contract/pull/3/files#r1286937171) doesn't address `storageIndex`.

I'm not going to insist though, it's a low impact change.